src/HybridSession.php

```diff
@@ -88,13 +77,7 @@ public function close()
         return true;
     }
 
-    /**
-     * @param string $session_id
-     *
-     * @return string
-     */
-    #[\ReturnTypeWillChange]
-    public function read($session_id)
+    public function read(string $session_id): string|false
```

Comment:
```suggestion
    public function read(string $session_id): string
```
This will never return false -  that would fail the `if ($data = $handler->read($session_id))` check, in which case an empty string gets returned at the end of the method.

Reply:
This maps to the return type on the [interface](https://www.php.net/manual/en/class.sessionhandlerinterface.php). 

Arguably, that last `return` after the `foreach` should be a `false`.  Don't necessarily want to go down the rabbit hole of debugging the fine print of this right now.

Using the more generic `string|false` gives us the flexibility to change afterwards if we want to.

Reply:
I suppose that's fine